### PR TITLE
Notification feature

### PR DESCRIPTION
**Pull request by kobakei**
NOTE: previous PR #92 has been destroyed because of my `git rebase` command ><
I have fixed feedbacks on the previous PR.

## Issue
- #82 

## Overview (Required)
- This PR provides first cut of notification feature.
  - Show notification 10 minutes ago of session which user checks.
  - Cancel notification when user unchecks session.
  - By tapping the notification, app will launch.
  - If user disables notification in setting screen, the notification will not show.

## Links
-

## Screenshot
Before | After
:--: | :--:
 | dump_4582993252243571689

### CODE REVIEW

**Comment by konifar** [comment]
100 kiriban!

**Comment by kobakei** [comment]
@konifar Thanks for your reviews.
I have fixed the reviewed points. Please check this PR :)

**Comment by konifar** [comment]
Sure! Thanks

**Comment by konifar** [comment]
LGTM! 💯 Thank you for your contribution!!

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+    <uses-permission android:name="android.permission.WAKE_LOCK" />
```
Comment (thagikura) [inline app/src/main/AndroidManifest.xml]: Sorry for jumping in. But was wondering if the WAKE_LOCK permission is required. Looks like no wake locks are obtained.

Comment (thagikura) [inline app/src/main/AndroidManifest.xml]: Hello @kobakei, @konifar,
Is the wake lock permission needed in the app?

Comment (kobakei) [inline app/src/main/AndroidManifest.xml]: Ah sorry, it may be unnecessary.
I will remove it.

Comment (konifar) [inline app/src/main/AndroidManifest.xml]: @thagikura Sorry, I overlooked your comment 🙇 
As you said, it looks unnecessary.

@kobakei If you have a time, could you please send PR again 🙏  Thanks!

Comment (thagikura) [inline app/src/main/AndroidManifest.xml]: Thank you!!

**Pull request merged**